### PR TITLE
Howitworks ui

### PR DESCRIPTION
**Pull request by lava10**
* how it works .tif hardcoded
* height and width set
* image properties changed to content from bundle resource

### CODE REVIEW

File: TeenTixMobileNEW/HowItWorksController.cs [hunk 1]
```diff
@@ -6,10 +6,33 @@
...
+
+		public override void ViewDidLoad ()
+		{
+			base.ViewDidLoad ();
+			Title = "How it Works!";
+			View.BackgroundColor = UIColor.White;
+
+			// a simple image
+			imageView = new UIImageView (UIImage.FromBundle ("howitworks_1.tif")); // Build Action:Content
+			imageView.Frame = new CGRect (327, 124, imageView.Image.CGImage.Width, imageView.Image.CGImage.Height);
```
Comment (thomasvandoren) [inline TeenTixMobileNEW/HowItWorksController.cs]: Do you want to calculate the x and y positions to accommodate different screen sizes?

**Comment by thomasvandoren** [comment]
LGTM ✨

File: TeenTixMobileNEW/HowItWorksController.cs (hunk 1 above)
Comment (lava10) [inline TeenTixMobileNEW/HowItWorksController.cs]: yes absolutely

**Comment by thomasvandoren** [comment]
Closing this. #44 subsumes it.

**Pull request closed without merging**